### DISCUSSION TITLE
How to declare getList function fields param for expanded collection?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2913] [Q&A]
I want to get name & contact no from members collection and plan name from plans collection and should neglect remaining fields. I understand that fields param should declare fields required to avoid getting all fields how to handle for expanded collection one level or in some case for more than one level
``` dart
final resultList = await pb.collection('members').getList(
 expand:"plans",
fields:"name, contactNo"
);
```

**Comment by ganigeorgiev** [comment]
The `fields` parameter follows the same structure as in the records json response, for example:
```js
fields: "id,created,name,contactNo,expand.plans.name,expand.plans.expand.someNestedRelField.id"
```

_Note that there is currently no support for wildcard (`*`) and you have to list all fields you want to return._
